src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleComputeDAO.scala

```diff
@@ -36,7 +36,7 @@ class HttpGoogleComputeDAO(appName: String,
 
   override implicit val service: GoogleInstrumentedService = GoogleInstrumentedService.Compute
 
-  override val scopes: Seq[String] = Seq(ComputeScopes.COMPUTE)
+  override val scopes: Seq[String] = Seq(ComputeScopes.CLOUD_PLATFORM)
```

Comment:
Should we be concerned about security implications?

Reply:
In [this PR](https://github.com/DataBiosphere/leonardo/pull/338) I gave `CLOUD_PLATFORM` scopes _only_ to `CloudResourceManager` (which does in fact need those scopes), and left `COMPUTE` as the default.

Reply:
We should probably replicate that in this PR; I can close mine if the changes are made here.

Reply:
Can this do similar to the above PR: only give `CLOUD_PLATFORM` to `CloudResourceManager` calls?